### DISCUSSION TITLE
[Solved] Endpoint not display in provider List

### DISCUSSION THREAD

**Discussion by magicdmer** [danny-avila/LibreChat#3679] [Troubleshooting] [answered]
### What happened?

When i upgrade to 0.7.4, i found the endpoint i add to librechat.yaml not display in provider list.

**Some of my configurations**：
![image](https://github.com/user-attachments/assets/5d5931b8-a54f-4175-ae2f-6408ad39d13c)

**Display**:
![IB4Z E($WPZB `W3MQ96VEM](https://github.com/user-attachments/assets/533f4ddc-6d9f-4f02-a4d9-fa5f5a7099fa)

And i can see the models i added in plugin selection.

### Steps to Reproduce

1. Add endpoint to librechat.yaml
2. Open librechat

### What browsers are you seeing the problem on?

Microsoft Edge

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
Check your startup logs, either viewing the terminal/container stdout or check the ./logs directory at project root. There was a breaking change in librechat.yaml

**Reply by magicdmer** [reply]
Yes, because the yaml content format has problems，the problem is solved after revision

**Reply by danny-avila** [reply]
thanks for confirming
